docs/app/pages/css/css-sections/icons/ux-icons/icon-preview/icon-preview.component.ts

```diff
@@ -21,7 +22,7 @@ export class IconPreviewComponent implements OnInit {
     @ViewChild(PopoverDirective, { static: true }) popover: PopoverDirective;
 
     ngOnInit(): void {
-        this.uxComponentSnippet = `<ux-icon name="${this.name}"></ux-icon>`;
+        this.uxComponentSnippet = `<ux-icon name="${this.name}" size="${this.iconSizeValue}"></ux-icon>`;
```

Comment:
Perhaps this size property should only appear whenever 16px is not selected, otherwise this is suggesting we add additional unused markup.

Reply:
Adding `size` to the code snippets was not part of this ticket, could you remove it altogether please? It's a nice idea but I think it will cause more problems than help.

Reply:
Yes, removed now.